Classes/Importer/AbstractImporter.php

```diff
@@ -316,12 +332,28 @@ protected function processBatch(NodeTemplate $nodeTemplate = null)
         $records = $this->preProcessing($records);
 
         array_walk($records, function ($data) use ($nodeTemplate) {
+            if (!\is_array($data)) {
```

Comment:
Just out of curiosity - is this a new standard to prefix php std methods with \?